features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Environments/Environments.jsx

```diff
@@ -48,6 +70,199 @@ const useStyles = makeStyles((theme) => ({
     saveButton: {
         marginTop: theme.spacing(3),
     },
+    shapeRec: {
```

Comment:
This file is tooooo long, Shall we please break this into several components ?

@tharikaGitHub , @Krishanx92 

and also shall we move out these style definitions from the component?